src/store/authors/actionCreators.js

```diff
@@ -0,0 +1,31 @@
+import {
+	ADD_COURSE_AUTHOR,
+	CREATE_AUTHOR,
+	REMOVE_COURSE_AUTHOR,
+	RESET_COURSE_AUTHORS,
+	SAVE_AUTHORS,
+} from './actionTypes';
+
+export const saveAuthors = (authors) => ({
```

Comment:
Please rename this method, you get the authors and not save them

Reply:
Got it